plugins/jobs/src/js/JobStopRun.js

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { componentFromStream } from "data-service";
+import { stopJobRun } from "#SRC/js/events/MetronomeClient";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/switchMap";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/startWith";
+import "rxjs/add/operator/catch";
+
+import JobStopRunModal from "./components/JobStopRunModal";
+
+function executeStop({ jobID, jobRun, onSuccess }) {
```

Comment:
can you please add JSDoc comments to all of these functions? I cant really follow the code below without types :(

Reply:
I gave it better function names, and I remove splat operations where I believe it is confusion.